app/src/main/java/com/leoart/koreanphrasebook/data/network/firebase/phrases/PhrasesRequest.kt

```diff
@@ -25,29 +26,37 @@ class PhrasesRequest : FireBaseRequest() {
         }
     }
 
-
-    fun getPhrases(categoryName: String): Observable<List<Phrase>> {
-        return Observable.create({ subscriber ->
-            dataBase.reference?.child("$CATEGORY_PHRASES/$categoryName")?.addListenerForSingleValueEvent(object : ValueEventListener {
+    fun getPhrases(): Observable<List<EPhrase>> {
```

Comment:
@greenhorn31 why request class returns local db entity?

Reply:
i dont know, it was changed by Lubomur